js/repeater.js

```diff
@@ -227,19 +312,17 @@
 			return markup;
 		},
 
-		disable: function() {
-			var disable = 'disable';
-			var disabled = 'disabled';
+		disable: function disable () {
 			var viewTypeObj = $.fn.repeater.viewTypes[this.viewType] || {};
 
-			this.$search.search(disable);
-			this.$filters.selectlist(disable);
-			this.$views.find('label, input').addClass(disabled).attr(disabled, disabled);
-			this.$pageSize.selectlist(disable);
-			this.$primaryPaging.find('.combobox').combobox(disable);
-			this.$secondaryPaging.attr(disabled, disabled);
-			this.$prevBtn.attr(disabled, disabled);
-			this.$nextBtn.attr(disabled, disabled);
+			this.$search.search('disable');
+			this.$filters.selectlist('disable');
+			this.$views.find('label, input').addClass('disabled').attr('disabled', 'disabled');
+			this.$pageSize.selectlist('disable');
+			this.$primaryPaging.find('.combobox').combobox('disable');
+			this.$secondaryPaging.attr('disabled', 'disabled');
+			this.$prevBtn.attr('disabled', 'disabled');
+			this.$nextBtn.attr('disabled', 'disabled');
```

Comment:
I think someone must have been worried about the extra keystrokes here. :-) Good call!